Source/Shaders/Appearances/PolylineColorAppearanceVS.glsl

```diff
@@ -25,4 +25,8 @@ void main()
     float angle;
     vec4 positionWC = getPolylineWindowCoordinates(p, prev, next, expandDir, width, usePrev, angle);
     gl_Position = czm_viewportOrthographic * positionWC;
+
+#ifdef LOG_DEPTH
+    czm_vertexLogZ(czm_modelViewProjectionRelativeToEye * p);
```

Comment:
Perhaps `czm_vertexLogDepth`

In general, stay consistent with "log depth" or "log z."

Also, it is OK to use log in all the private API, but use logarithm in the public API unless the technique is commonly called "log", then just keep the public API with log.